app/src/main/java/com/devshish/internship/presentation/ui/web/WebFragment.kt

```diff
@@ -24,19 +26,33 @@ class WebFragment : Fragment(R.layout.fragment_web) {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
-        with(binding) {
-            webView.apply {
-                settings.javaScriptEnabled = true
-                webViewClient = client
-                loadUrl(args.link)
-            }
+        (activity as AppCompatActivity).supportActionBar?.apply {
+            setTitle(R.string.auth_sign_in_with_genius)
+            setHomeAsUpIndicator(R.drawable.ic_close)
+            setDisplayHomeAsUpEnabled(true)
         }
 
+        removeCookies()
+        setupWebView()
```

Comment:
I think you can move code from removeCookies to setupWebView method - because it is part of setup of web view